### DISCUSSION TITLE
Is it possible to install multiple versions of a tool and switch between them?

### DISCUSSION THREAD

**Discussion by MunifTanjim** [mason-org/mason.nvim#195] [Q&A]
Say some project requires a specific version of a formatter, will it be possbile to switch between formatter version using mason?

**Comment by AlixShahid** [comment]
No. it would be nice to have this feature but implementing it would make Mason more complex.

**Reply by zachschultz** [reply]
I hear you on the complexity, I just wanted to chime in that I'd love to see this in the future (I have a lot of different workspaces which require different tool versions)
